athlinks-schema.ttl

```diff
@@ -14,6 +14,12 @@
 
 ### CLASSES
 
+athlinks:User 
```

Comment:
This should probably be modeled using an foaf:account of if we can find a schma.org/Account.